index.bs

```diff
@@ -3726,6 +3788,38 @@ SHOULD be aborted.
     See [WHATWG HTML WG Issue #2711](https://github.com/whatwg/html/issues/2711) for more details.
 
 
+## WebAuthn Interfaces ## {#sctn-interfaces}
+
+The subection below defines custom interfaces used throughout WebAuthn.
+
+### Custom WebAuthn Exceptions ### {#iface-custom-webauthn-exceptions}
+
+For descriptions of these exceptions,
+please see [[#sctn-create-request-exceptions]] and [[#sctn-get-request-exceptions]].
+
+<xmp class="idl">
+[Exposed=Window, Serializable]
+interface HybridPrerequisitesError : DOMException {
+};
+
+[Exposed=Window, Serializable]
+interface TimeoutError : DOMException {
+};
+
+[Exposed=Window, Serializable]
+interface UserCancellationError : DOMException {
+};
+
+[Exposed=Window, Serializable]
+interface UserHybridCancellationError : DOMException {
+};
+
+[Exposed=Window, Serializable]
+interface UserVerificationError : DOMException {
+};
+</xmp>
+
+
```

Comment:
@emlun I've defined custom interfaces for the new errors so they're derived from `DOMException`. Thoughts?

Reply:
Replied in https://github.com/w3c/webauthn/pull/2095#pullrequestreview-2208153548